include/NAS2D/Resources/Sound.h

```diff
@@ -25,11 +25,10 @@ class Sound: public Resource
 public:
 
 	Sound() = default;
-	Sound(const Sound& other) = default;
```

Comment:
The merge commit with `master` removes the `Sound` copy constructor. This looks to be accidental. The conversion constructor from `std::string` was moved, so perhaps that's related to how this happened.